### PR TITLE
Adds the TRIAGING.md file to outline our triaging process

### PR DESCRIPTION
**Pull request by dlvenable**
### Description

This PR adds the `TRIAGING.md` file which we will use to guide our existing triage meetings. This is part of our goal of making our triage meetings open to the community.
 
### Issues Resolved

N/A
 
### Check List
- [ ] New functionality includes testing.
- [ ] New functionality has a documentation issue. Please link to it in this PR.
  - [ ] New functionality has javadoc added
- [ ] Commits are signed with a real name per the DCO

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/data-prepper/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: TRIAGING.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+All new issues are triaged weekly.
+
+### What happens if my issue does not get covered this time?
+
+Each meeting we seek to address all new issues.
+However, should we run out of time before your issue is discussed, you are always welcome to attend the next meeting or to follow up on the issue post itself.
+
+### How do I join the triage meeting?
+
+Meetings are hosted regularly at 2:30 PM US Central Time (12:30 PM Pacific Time) and can be joined via the links posted on the [OpenSearch Meetup Group](https://www.meetup.com/opensearch/events/) list of events.
```
Comment (graytaylor0) [inline TRIAGING.md:19]: I guess we don't want to commit to a day int he README? We just use the event schedule for that?

**Review by graytaylor0 — APPROVED** [review]
This looks good!

File: TRIAGING.md (hunk 1 above)
Comment (dlvenable) [reply TRIAGING.md:19]: That is a miss. I will update it to include Tuesday as the day.

**Comment by krisfreedain** [comment]
@dlvenable - this looks excellent. Well done.

**Review by KarstenSchnitter — APPROVED** [review]
Looks good to me.

**Review by graytaylor0 — APPROVED** [review]

**Pull request merged**